src/public/stageView.ts

```diff
@@ -0,0 +1,64 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { SdkError } from '../public';
+import { FrameContexts } from './constants';
+
+/**
+ * Namespace to interact with the stage view specific part of the SDK.
+ */
+export namespace stageView {
+  /**
+   * Parameters to open a stage view.
+   */
+  export interface StageViewParams {
+    /**
+     * The application ID of the Teams application to be opened.
+     */
+    appId: string;
+
+    /**
+     * The URL of the content to display.
+     */
+    contentUrl: string;
+
+    /**
+     * The chat or channel ID.
+     */
+    threadId: string;
+
+    /**
+     * The title to give the stage view.
+     */
+    title: string;
+
+    /**
+     * The Teams app website URL.
+     */
+    websiteUrl?: string;
+
+    /**
+     * The entity ID of the Teams app.
```

Comment:
This should say something like "The entity ID of the Teams app content being opened."